dwave/optimization/symbols.pyx

```diff
@@ -1554,40 +1534,71 @@ cdef class ListVariable(ArraySymbol):
         # The validity of the state is checked in C++
         self.ptr.initialize_state(self.model.states._states[index], move(items))
 
-    # An observing pointer to the C++ ListNode
-    cdef cppListNode* ptr
+    def upper_bound(self):
+        """The highest value the integer(s) can take (inclusive)."""
```

Comment:
```suggestion
        """The highest value allowed for the integer symbol."""
```

Reply:
Haha, this is your docstring! Or at least not mine. But sure.